src/transformers/modeling_longformer.py

```diff
@@ -570,7 +591,13 @@ def _compute_global_attn_output_from_hidden(
         global_value_vectors = self.value_global(hidden_states)
 
         # normalize
-        global_query_vectors_only_global /= math.sqrt(self.head_dim)
+        global_query_vectors_only_global /= torch.sqrt(
+            torch.tensor(
+                self.head_dim,
+                device=global_query_vectors_only_global.device,
+                dtype=global_query_vectors_only_global.dtype,
+            )
+        )
```

Comment:
Out of curiosity, is the creation of a tensor + pytorch operation faster than the previous math operation on a tensor ?

Reply:
Thanks for noting that, I actually didn't time it and thought it nice to get rid of the math library. 
From these tests on a gpu: https://colab.research.google.com/drive/1q830_JmCLyxmlQCTBxe3Rj2G3pZuniCY?usp=sharing it seems that it's actually much slower to first create a tensor and do everything in PyTorch (it's much worse apparanly in TF) => so using math should be preferred over tensor creation I guess and I have to update my beliefs about "doing everything in tensors a bit" :+1: 

Is this obvious to you @LysandreJik that using `math` is faster? I would have assumed that under the hood a tensor is created anyways....do you have an idea why `math` is faster?

Reply:
No it isn't obvious to me at all! I would have thought that dividing a tensor by a python value would create a tensor for the python value, resulting in the same number of operations. I think it's cleaner to only use torch like functions too, but from your benchmarking it seems that it's not always the case! 

Thanks for diving into it, really useful!